types/streamify-string/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "esModuleInterop": true,
```

Comment:
why do we need `esModuleInterop` to be `true` for this package?

Reply:
I added this to be consistent with the [streamify-array](https://github.com/rubensworks/streamify-array.js#readme) type definition package.

To my understanding, it is necessary because the streamify-array package is commonjs and we want to be able to import it into an es6 module.

The other option would be to remove this and make it `export default streamifyArray` in index.d.ts and then remove this, however, I don't know if this is good practice, and whether there would be compatibility issues if trying to import into a project using commonjs.

Reply:
it's a no-no on DT (and yes, you need a reason to override this):
https://github.com/DefinitelyTyped/DefinitelyTyped#esmoduleinteropallowsyntheticdefaultimports
it's a way-to-go when consuming packages (and default when creating tsconfig)